src/containers/SortingView.tsx

```diff
@@ -97,6 +99,32 @@ const SortingView: React.FunctionComponent<Props> = (props) => {
     }
   }, [onSetExternalUnitMetrics, setExternalUnitMetricsStatus, externalUnitMetricsStatus, sorting, sortingId, hither])
 
+  const [recordingInfoStatus, setRecordingInfoStatus]= useState<'waiting' | 'calculating' | 'finished' | 'error'>('waiting')
```

Comment:
Retrieve the recording info via hither if not already present on the recording.

Reply:
I agree, I think we should uniformize this as CalculationStatus or something. With typescript I think we can get away with

type CalculationStatus = 'waiting' | 'calculating' | ...

using strings, because the ts compiler will force correct spelling, etc.